starter-code/public/javascript/index.js

```diff
@@ -1,23 +1,77 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
-
+//const charactersAPI = new APIHandler('http:/ / localhost: 8000');
 window.addEventListener('load', () => {
   document.getElementById('fetch-all').addEventListener('click', function (event) {
+    axios.get("http://localhost:8000/characters").then((result) => {
+      let resultsFromApi = result.data;
+      console.log(resultsFromApi);
 
+      resultsFromApi.forEach((character) => {
+        let newNode = document.getElementsByClassName("character-info")[0].cloneNode(true)
+        newNode.getElementsByClassName("name")[0].innerText = character.name
+        newNode.getElementsByClassName("occupation")[0].innerText = character.occupation
+        newNode.getElementsByClassName("weapon")[0].innerText = character.weapon
+        newNode.getElementsByClassName("cartoon")[0].innerText = character.cartoon
+        document.getElementsByClassName("characters-container")[0].append(newNode)
+      });
+    });
```

Comment:
its also good to do some error handling
this will help you in project module to find your error faster and easie
, so here you should write

.catch(error => console.log("something with grtting charecter id went wrong", error))